### PR TITLE
chore: optimize cmake

### PR DESCRIPTION
**Pull request by asterwyx**
Optimize cmake variables.

Log: optimize cmake

### CODE REVIEW

**Review by wineee — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -16,23 +16,26 @@ project(DtkSystemSettings
...
 endif()
 
-set(LIB_NAME dtk${DTK_VERSION_MAJOR}systemsettings)
-set(DtkSystemSettings Dtk${DTK_VERSION_MAJOR}SystemSettings)
-set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake" ${CMAKE_MODULE_PATH}) # Set module path for include() command search
-
-include(DtkUtils) # Include cmake/DtkUtils.cmake
+set(LIB_NAME dtk${VERSION_SUFFIX}systemsettings)
+set(DtkSystemSettings Dtk${VERSION_SUFFIX}SystemSettings)
+set(DTK_INCLUDE_INSTALL_DIR "include/dtk${DTK_VERSION_MAJOR}/DSystemSettings" CACHE STRING "Include directory headers")
```
Comment (wineee) [inline CMakeLists.txt]: include 换成 ${CMAKE_INSTALL_INCLUDEDIR} 更好

**Review by wineee — APPROVED** [review]

**Review by 18202781743 — DISMISSED** [review]

**Comment by asterwyx** [comment]
Do not merge this one!!! There are still some problems in building a deb package.

**Review by ComixHe — APPROVED** [review]

**Pull request merged**